index.html

```diff
@@ -124,7 +124,7 @@ <h3>What Users Are Saying</h3>
 					<br/><br/>
 					</span>
 					<h3>Developers</h3>
-					<p>Todo.txt CLI and Todo.txt Touch are proudly open source. Browse the source code for the <a href="https://github.com/todotxt/todo.txt-cli">CLI</a>, <a href="https://github.com/todotxt/todo.txt-touch-ios">iOS</a>, and <a href="https://github.com/todotxt/todo.txt-touch">Android</a>.</p>
```

Comment:
@1Maxnet1 Why was the CLI link removed?

Reply:
To be honest, i don't know, why i should have done this. Mistake of mine. I updated the other links in the line to work again and brought back CLI-Link. Hopefully it's ready to get merged now. Sorry for the chaotic PR.